infrastructure/tooling/src/generate/generators/docker-compose.js

```diff
@@ -58,7 +59,7 @@ const defaultValues = {
   NODE_ENV: 'development',
 
   DEBUG: '',
-  LEVEL: 'warning',
+  LEVEL: 'info',
```

Comment:
Switching again to a slightly more verbose mode 🙈 Having some logs is better than no logs, as it feels like nothing is happening